plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -203,29 +203,25 @@ public void start() {
     public void listPlugins() {
         if (cfg.isShowPluginsToBeDownloaded()) {
             System.out.println("\nInstalled plugins:");
-            for (Map.Entry<String, VersionNumber> installedPlugin : installedPluginVersions.entrySet()) {
-                System.out.println(installedPlugin.getKey() + ": " + installedPlugin.getValue());
-            }
+            installedPluginVersions.entrySet().stream()
+                    .sorted(Map.Entry.comparingByValue())
+                    .forEach(p -> System.out.println(p.getKey() + " " + p.getValue()));
```

Comment:
you could probably extract this to a method as it's called 3 times and then use something like:
```suggestion
                    .forEach(this::logPlugin);
```